incubator/node-red/image/Dockerfile-stack

```diff
@@ -0,0 +1,30 @@
+FROM node:10
+
+ENV APPSODY_MOUNTS=/:/project/user-app
+ENV APPSODY_DEPS=/project/user-app/node_modules
+
+ENV APPSODY_WATCH_DIR=/project/user-app
+ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/node_modules
+ENV APPSODY_WATCH_REGEX="^.*.json$"
+
+ENV APPSODY_INSTALL="npm install --prefix user-app && npm audit fix --prefix user-app"
```

Comment:
`APPSODY_INSTALL` --> `APPSODY_PREP` as `APPSODY_INSTALL` has been deprecated

Reply:
Where can I find out what the current valid env vars are? This stack is only a month old and I don't follow every change being made to know what has been deprecated and what hasnt.

Reply:
Our stack env vars are documented here: https://appsody.dev/docs/stacks/environment-variables

Reply:
So I just need to rename the env var? It's purpose hasn't changed? I note the nodejs-express and nodejs stacks still use `APPSODY_INSTALL`.

Reply:
Yes it is just renaming, the purpose has not changed. (Well spotted with the node stacks - we will try to remember to update the env var in the new stack releases!)